catalog/publishers.jsonld

```diff
@@ -0,0 +1,29 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@graph": [
+    {
+      "@id": "http://www.getty.edu/research/",
+      "identifier": "Getty"
+    },
+    {
+      "@id": "https://www.cultureelerfgoed.nl",
+      "identifier": "RCE"
+    },
+    {
+      "@id": "https://www.wereldculturen.nl",
+      "identifier": "Getty"
```

Comment:
I think 'Getty' should be replaced by 'NMvW' (or 'Wereldculturen' - its abbreviation may not be known by people)?

Reply:
Decided to use NMVW (capital ‘V’ as that’s [how they write it themselves](https://www.tropenmuseum.nl/nl/over-het-tropenmuseum/stichting-nationaal-museum-van-wereldculturen)) for now. For API users, we have the IRI for dereferencing (although it resolves to HTML not RDF in most cases) and finding the name and details of the organization. Something similar may be nice in the demonstrator as well, but that’s out of scope for now.